crypto/ocsp/ocsp_lib.c

```diff
@@ -129,6 +129,11 @@ int OCSP_id_cmp(const OCSP_CERTID *a, const OCSP_CERTID *b) {
 
 int OCSP_parse_url(const char *url, char **phost, char **pport, char **ppath,
                    int *pssl) {
+  if (url == NULL) {
```

Comment:
Is there something unique about url? Should we check all the pointers for null?

Reply:
Yeah we should check all pointers. I just did a test and assigning the other values `null` will result in a memory failure.